### ISSUE TITLE
MoA studies via the generation of resistant mutants 

### ISSUE THREAD

**Issue by mbhebhe** [OpenSourceMalaria/Series3#12]
[Last year](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/524), we sent OSM-S-106 to the Winzeler Lab (UCSD) for mechanism of action studies via the generation of resistant mutants. Last week they emailed us the results! **Exciting!!**
Raw data is available [here](https://au-mynotebook.labarchives.com/share/Tha%2520OSM%2520Series%25203/NDA5LjV8MjcwNjkvMzE1L1RyZWVOb2RlLzE2OTE2ODYyNjd8MTAzOS41). 

This was they said in the email:

"Results for OSM-S-106 are a little more complicated but there is some consistency across the 3D7 and Dd2 results.
 
In the 3D7 line, PF3D7_0103200 (nucleoside transporter 4) has a His26Gln mutation and PF3D7_0802000 (glutamate dehydrogenase, putative) has a Lys64Gln mutation in all clones. There may also be slight amplification across PF3D7_1107800 (AP2 domain transcription factor, putative). I’ve noted that the resistant clones all showed a ~3x shift so the consistency of these mutations makes sense.
 
In the Dd2 line, there are also mutations in PF3D7_0103200 (nucleoside transporter 4), but they are different across flasks 2 and 3 – His320Leu in flask 2 clones and Ser22Cys in flask 3 clones. An Asp200Tyr mutation in PF3D7_0802000 (glutamate dehydrogenase, putative) is in both clones from 3 but not 2. Interestingly, both clones from flask 2 have an Arg487Ser mutation in PF3D7_0211800 (asparagine--tRNA ligase), while the clones from flask 3 have an amplification across this region but not the SNV. There is a much more significant difference in the fold-shifts for the resistant lines (~9x in flask 2 clones vs. ~2-3x in flask 3 clones) so the SNV vs. CNV may account for that discrepancy. I think the mostly likely explanation of these results is that asparagine--tRNA ligase is the target and nucleoside transporter 4 and glutamate dehydrogenase mutations are resistance/compensatory mechanisms (though glutamate dehydrogenase is also a possible Pf target)."

**In short, asparagine-tRNA ligase is the target.**

------------------------------------------------------------------------------------------------------------------

I have been doing a bit of reading on asparagine tRNA ligase/synthetase (AsnRs).
From what I have read this is novel Pf target and I could not find any inhibitors for this target. We asked the Winzeler Lab if they knew of other molecules that hit this enzyme. They don't know of any other molecules but they offered to do more target validation through CRISPR/Cas9 and conditional knock down experiments. (Is this a good idea?).

------------------------------------------------------------------------------------------------------------------
Since this is a novel target, it seems that this Pf enzyme has only been expressed/cloned once (Ref 1). Please let us know if you know of anyone that has worked on Pf AsnRs or has it. We would like to do some binding studies on this enzyme. Conducting binding studies on Homo sapien AsnRs, would also be great . On the protein data bank website there are 8 AsnRs structures from 4 organisms (one human and no Pf). 

Asparagine tRNA synthetase catalyses the attachment of Asn to its cognate tRNA to produce tRNA<sup>Asn</sup>, a substrate for translation. This is done via a 2 step reaction. An ATP molecule and Asn molecule enter the active site, forming an aminoacyl-adenylate intermediate followed by the esterification of the Asn to the 30 end of the tRNA, forming the final tRNA<sup>Asn</sup> (Ref 2). 

The papers I have read so far talk about Pf proteome having a lot of asparagine rich regions but yet there is only one gene that encodes tRNA<sup>Asn</sup>. They conducted studies to see if there is any correlation between these asparagine rich regions and tRNA<sup>Asn</sup>. 
AsnRS is highly expressed in the cell and it is not the limiting 'reagent'  in the first step of the asparaginylation reaction (Ref 1&3). 

-------------------------------------------------------------------------------------------------------------------

Stuart Ralph and colleagues from UniMelb wrote a review on Aminoacyl-tRNA synthetases as drug targets in eukaryotic parasites. In this review they say that disrupting tRNA synthetases will poison the process of protein translation and this will affect the parasite. Parasites have bacteria like protein translation pathways and are different to those of human pathways. We can target the binding site for ATP, amino acid and the fold for tRNA<sup>amino acid</sup> recognition and binding (Ref 2). 
In summary, aminoacyl-tRNA synthetases are promising drug targets and if OSM-S-106 truly does target the novel, Pf AsnRs then we are have a promising drug.

Ref 1: doi: 10.1074/jbc.M113.522896
Ref 2: http://dx.doi.org/10.1016/j.ijpddr.2013.10.001
Ref 3: doi: 10.1093/femsre/fux046

Sorry for the text overload! Not even a single diagram :'(

**Comment by holeung** [comment]
I can work on a homology model and try to identify an OSM-S-106 binding site. I can also produce Pf asparagine-tRNA ligase in the lab, verify binding to OSM-S-106, and pursue a co-crystal structure.

**Comment by drc007** [comment]
Excellent

**Comment by holeung** [comment]
Looks like the structure of Pf has already been done, [PDB 5JLD](http://www.rcsb.org/structure/5JLD). The work was published in 2016, "[Dimerization of Arginyl-tRNA Synthetase by Free Heme Drives Its Inactivation in Plasmodium falciparum](http://doi:10.1016/j.str.2016.06.018)".

**Comment by MFernflower** [comment]
@holeung Would you be able to dock the drug to the PDB? I'd bet our drug is binding to the ATP intake site on the protein possibly causing a conformation change that promptly deactivates said enzyme

**Comment by MFernflower** [comment]
@holeung Just realized the PDB entry is for the malarial **arginine** ligase - perhaps it could still be useful for binding study as it could have the same atp intake site?

**Comment by holeung** [comment]
Right you are. Thanks for pointing that out. I need to use my reading glasses more.

**Comment by MFernflower** [comment]
Just updated the wiki with a link back to this GHI @mbhebhe @mattodd

**Comment by mbhebhe** [comment]
A homology model would be great. @holeung how easy/difficult will it be to produce Pf asparagine-tRNA ligase?
@MFernflower I had the same thought about OSM-S-106 binding to the ATP binding site. Thanks for updating the wiki.

**Comment by MFernflower** [comment]
Does not seem too difficult no glycosylation - just order an arabad plasmid with the pfal sequence from a plasmid builder service and insert into TOP10  E. Coli 

Purification and co crystal formation is what takes time 

@mbhebhe

**Comment by MFernflower** [comment]
Also @mbhebhe if it is a mimic of ATP the purine analogue of OSM-S-106 should be somewhat more potent yes?

**Comment by drc007** [comment]
> Purification and co crystal formation is what takes time

and we should not under estimate the length of time it can take.

**Comment by holeung** [comment]
I will start looking into the logistics of producing the Pf AsnRS protein. It shouldn't be too difficult, but it requires time, money, and labor, which are always challenges. It could take 3 months if everything goes smoothly.

**Comment by mbhebhe** [comment]
@holeung for producing the homology model! 
Should we wait for the target validation studies and conditional knock down experiments to be done first before trying to produce the Pf AsnRS protein? I will be sending them more compound to them by end of this week

**Comment by mbhebhe** [comment]
@holeung they said validation studies through CRISPR/Cas9.

**Comment by holeung** [comment]
@mbhebhe, great. I will contact them directly so we can coordinate and avoid duplication of efforts.

**Comment by mbhebhe** [comment]
Its actually people from the Wellcome Sanger Institute in England. I will ask them to get a GH account so that we can easily communicate on here

**Comment by MFernflower** [comment]
I apologize for assuming it was the Winzeler laboratory! @mbhebhe @holeung

**Comment by holeung** [comment]
@mbhebhe , thank you for the clarification. Nice that we have so many contributors!

**Comment by MFernflower** [comment]
@holeung Was just wondering if you have access to poseview to create a 2d image of the docking study?

**Comment by holeung** [comment]
@MFernflower , 2D diagram attached. Made using Schrodinger Maestro.

![docked_2d](https://user-images.githubusercontent.com/10951570/50870762-05c3c080-137f-11e9-88bc-47481ddb00f8.png)

**Comment by MFernflower** [comment]
Rather interesting - It would explain some of the sensitivity to substitution that has been occurring - Going off of this it seems like thieno(2,3‐c)pyridines would be tolerated

**Comment by MFernflower** [comment]
@mbhebhe @holeung Gives me an idea:
![129o3j3](https://user-images.githubusercontent.com/3164942/50871697-77edd280-138a-11e9-8323-8374a6d1181f.png)


NC=1N=CC=C2C1SC(=C2)C=2C=C(C=CC2)S(=O)(=O)N
IPIRNZZMWQZXIW-UHFFFAOYSA-N

**Comment by drc007** [comment]
Still looks like replacing the sulphonamide with carboxylic acid should work.

**Comment by MFernflower** [comment]
@drc007 perhaps the reason the acid was not potent was because of cell penetrance?

**Comment by sottilie** [comment]
Hi all, just to connect you all to the MalDA consortium and explaining who is doing what. I am the Program Manager overseeing all the projects and coordinating the multiple groups (14 in total) in their efforts. We received your compound OSM-S-106 last year and I asked Eva Istvan from the Goldberg lab to perform the in vitro selections. Once she had isolated resistant clones the Winzeler lab performed the sequence analysis and identified the potential target. We have now reached out to Marcus Lee to validate the target by CRISPR/Cas9 and Jacquin Niles for conditional knockdown studies. We are all excited to be part of this group and working with you. Please let us know how we can help. Best, Sabine You can also reach me by email: sottilie2@gmail.com

**Comment by holeung** [comment]
@sottilie , thanks for the intro post! After the validation tests, I am interested in following up by biophysics and crystallography. Or do you already have that covered?

**Comment by sottilie** [comment]
No I don't thing we will be able to do biophysics and crystallography - so wonderful if you can follow up with this. @holeung

**Comment by mattodd** [comment]
Great thread based on a fascinating result. My guess is that you'd want to wait on the knockdowns before investing any experimental time @holeung ?

@mbhebhe have we not made the CH analog @MFernflower suggests? The thiopyridine?

@holeung the pose of the ATP in the active site - how confident are you of the pose? I'd thought that in kinases (at least) the methionine residue (which you have) would be pointing at the free NH2? i.e. flipped over from what you have. Which is not to say that that applies here.

**Comment by cdsouthan** [comment]
Shortest path would be to find someone running the assays for this enzyme in vitro?   Try the authors of https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3868750/ and https://www.ncbi.nlm.nih.gov/pubmed/24196969

**Comment by sottilie** [comment]
Charise in Jacquin Niles' lab has received some of your compound and is already starting experiments on her cKD cell line.

**Comment by MFernflower** [comment]
@mbhebhe @mattodd I'd also be curious what effect the thieno[2,3‐c]pyridine system has on real world water solubility - IIRC the S3 hit had poorer than expected water solubility due to strong hydrogen bonds between molecules of OSM-S-106

**Comment by mbhebhe** [comment]
We have not made the thieno[2,3‐c]pyridine analogue. The precursor is expensive though.  
![image](https://user-images.githubusercontent.com/26862286/51218898-7b80eb00-1982-11e9-93a6-8031a8e4af87.png)

**Comment by MFernflower** [comment]
There are many synthetic methods reported eg
https://doi.org/10.1016/j.bmcl.2006.08.041

>

**Comment by MFernflower** [comment]
@mbhebhe Sigma also happens to sell this: https://www.sigmaaldrich.com/catalog/product/aldrich/728780

**Comment by MFernflower** [comment]
Please excuse my mistake regarding the Sigma Aldrich product

**Comment by StuartRalph** [comment]
There are a couple of other straightforward ways of gathering validation data that I can think of.
First is that if asparagine tRNA synthetase is the target, the compound ought to block protein synthesis. We could test that relatively easily by radioisotope labelling of amino acids in the presence and absence of the inhibitor.
Second is that the parasites may be less susceptible to the inhibitor in the presence of higher concentrations of asparagine, as the tRNA synthetase may be more efficient in the presence of higher asparagine levels. This may not be as pronounced if the inhibitor is indeed binding only to the ATP binding pocket, but is nonetheless a very easy experiment to conduct. Our lab would be happy to help out with either of those experiments if you are interested in going ahead with them. Stuart.

**Comment by MFernflower** [comment]
I think the 14C-asparagine experiment would be super useful - Thoughts @mattodd and @mbhebhe and @drc007  ???

**Comment by StuartRalph** [comment]
It wouldn't have to be 14C-asparagine - although this seems more direct, if you get inhibition of protein synthesis, it would likely impact all amino acids, so you could use whatever radio-labeled amino acid you have handy

**Comment by sottilie** [comment]
The Winzeler lab is also able to do a protein synthesis inhibitor assay
since we have another aa RS target for a different compound. Only problem
we are out of your compound OSM-106.

Sabine Ottilie, Ph.D.
Project Manager
Malaria Drug Accelerator (MalDA) Consortium
Yeast Drug Target Discovery
Lab of Prof. E. Winzeler | UC San Diego
Division of Host-Microbe Systems & Therapeutics
Biomedical Research Facility 2 | 5220D
9500 Gilman Dr | MC 0760 | La Jolla, CA 92093-0760
http://winzeler.ucsd.edu/

**Comment by MFernflower** [comment]
@StuartRalph This might be an obvious question but what happens when the ribosome runs out of a certain trna - does translation just lock up on that codon or does the chain prematurely terminate?

**Comment by StuartRalph** [comment]
> Great thread based on a fascinating result. My guess is that you'd want to wait on the knockdowns before investing any experimental time @holeung ?

Not 100% clear on what you are expecting from the knockdown experiments. This is almost certainly an essential protein, and the piggybac data indicate it can't be disrupted. The parasite will have reduced protein synthesis and may be more susceptible to the inhibitor, but that might also apply to knockdown of other off-target proteins required for protein synthesis. (eg if you knockdown an off target ribosomal component, protein translation will be compromised, and those parasites may also be more sensitive to a second (chemical) insult to translation. I would have thought that induced or constitutive over-expression of the target (with or without the SNPs) would be a more direct and more conclusive genetic experiment.

**Comment by sottilie** [comment]
You would see a shift in IC50 if the compound indeed hits the target. It's
a quick validation study we do for all our putative 'targets' that we find
by in vitro selections.

Sabine Ottilie, Ph.D.
Project Manager
Malaria Drug Accelerator (MalDA) Consortium
Yeast Drug Target Discovery
Lab of Prof. E. Winzeler | UC San Diego
Division of Host-Microbe Systems & Therapeutics
Biomedical Research Facility 2 | 5220D
9500 Gilman Dr | MC 0760 | La Jolla, CA 92093-0760
http://winzeler.ucsd.edu/

**Comment by MFernflower** [comment]
@sottilie But would it be possible to obtain reliable results when the target is an essential piece of machinery for protein synthesis?

**Comment by sottilie** [comment]
Jacquin Niles has done this for quite a few aaRS and it seems to work well.

**Comment by MFernflower** [comment]
Extremely interesting....

**Comment by mbhebhe** [comment]
@StuartRalph I can send some compound to you guys. Is 5mg enough?

> There are a couple of other straightforward ways of gathering validation data that I can think of.
> First is that if asparagine tRNA synthetase is the target, the compound ought to block protein synthesis. We could test that relatively easily by radioisotope labelling of amino acids in the presence and absence of the inhibitor.
> Second is that the parasites may be less susceptible to the inhibitor in the presence of higher concentrations of asparagine, as the tRNA synthetase may be more efficient in the presence of higher asparagine levels. This may not be as pronounced if the inhibitor is indeed binding only to the ATP binding pocket, but is nonetheless a very easy experiment to conduct. Our lab would be happy to help out with either of those experiments if you are interested in going ahead with them. Stuart.

**Comment by mbhebhe** [comment]
@sottilie I will send some more compound tomorrow

**Comment by cdsouthan** [comment]
JFTR , assuming  https://pubchem.ncbi.nlm.nih.gov/compound/44528665,  MQMXDJVOZKMSNT-UHFFFAOYSA-N and OSM-S-106 are one and the same :)  PubChem provides a useful list of 33 analogues https://www.ncbi.nlm.nih.gov/pccompound?cmd=Link&LinkName=pccompound_pccompound&from_uid=44528665.  This represents the 90% Tanimoto shell as computed by PubChem.  Note that other OSMs are in their (via ChEMBL) and that 18 should be available via credit card (OK, I know OSM doesn't have one).  
![capture](https://user-images.githubusercontent.com/3713939/51244549-a91b6380-1985-11e9-93d8-a25cccfcb27f.JPG)

**Comment by sottilie** [comment]
Many thanks! You can use the address below.

Sabine

Sabine Ottilie, Ph.D.
Project Manager
Malaria Drug Accelerator (MalDA) Consortium
Yeast Drug Target Discovery
Lab of Prof. E. Winzeler | UC San Diego
Division of Host-Microbe Systems & Therapeutics
Biomedical Research Facility 2 | 5220D
9500 Gilman Dr | MC 0760 | La Jolla, CA 92093-0760
http://winzeler.ucsd.edu/

**Comment by MFernflower** [comment]
@cdsouthan OSM credit card makes my brain think of some sort of custom VisaCard with a giant mosquito in the center

**Comment by MFernflower** [comment]
@mbhebhe @mattodd Just added the Thieno[2,3‐c]pyridine analogue to the DCNYS section

**Comment by holeung** [comment]
I think we should wait for some of the results of the knockdown and biochemical assays before we move ahead. We can perform biophysical binding assays easily, IF the protein is easy to produce. Crystallography will likely require months of work.

**Comment by MFernflower** [comment]
@holeung Indeed, hopefully the panels provided by @sottilie and @StuartRalph will shine light on this drug's MOA without having to resort to drastic measures like crystallography!

**Comment by drc007** [comment]
Just posted on my website details of MMV call for proposals https://www.cambridgemedchemconsulting.com/news/index_files/ab9b8d1802e570728c7b25a9787aa115-358.html in particular there is support to investigate the mechanism of action of compounds.

**Comment by mattodd** [comment]
OK, great to have all this excellent input here. The data from the MalDA consortium are suggestive of an interesting new mechanism of action (Pf asparagine tRNA ligase), but we should not become too excited until we've more supporting data. To that end, we have several ongoing lines of inquiry:

a) Marcus Lee (Sanger) is doing CRISP/Cas9 validation and Jacquin Niles (MIT) is doing conditional knock-downs.

b) @StuartRalph (Uni Melbourne) is happy to investigate by using radiolabelled amino acids +/- OSM-S-106 to see if protein synthesis is affected (as predicted), and to see if the IC50 alters in the presence of excess asparagine - I'm hopefully going to be able to meet Stuart next week in Melbourne as it happens. The former experiment is something that @sottilie is also happy to do (and I think may have started?). Either or both would be very good. *Unnecessary duplication* is to be avoided, but there is value in similar experiments being performed in two places if that gives stronger evidence of a conclusion.

c) @holeung has built a homology model of the target, and has started docking of ligand and target. He is happy to express the protein in his lab if helpful towards a biophysics or co-crystal structure validation, once the above experiments provide more evidence that we're looking at the right target. I understand from talking to Al Edwards that the SGC would also be willing to express the protein (in Brazil, I believe) which may work well given they have ample experience in ligases (or synthetases as they're also known). tbd.

So it would seem sensible to proceed with a) and b) now, and await those results before embarking on c)? Have I missed anything?

As new data come in we can wind up this Issue, but still lots of things to take care of here first. @mbhebhe take a look at the commercially-available structures @cdsouthan identified and see if there any we can buy. Maybe compile a new Chemdraw on a new issue so we can see prices of any that are ready to purchase and we can decide? This commercial search is worth re-doing every now and again.

**Comment by drc007** [comment]
@mattodd "Have I missed anything?" 

Perhaps check OSM-S-106 in a carbonic anhydrase assay?

**Comment by mattodd** [comment]
@drc007 indeed - a loose end captured in the [wiki](https://github.com/OpenSourceMalaria/Series3/wiki/Mechanism-of-Action), but where we still need experimental effort, right @mbhebhe ? (I was here just trying to summarise the present discussion in this issue, though)

**Comment by cdsouthan** [comment]
How many times has OSM-S-106 been re-synthesized and re-tested in // with the original batch by the same assay operation?  AWAK it is somewhere between prudent and crucial to do this

**Comment by drc007** [comment]
@mattodd @MFernflower @mbhebhe @cdsouthan CA inhibition is both a possible mode of action and a potential undesirable side-effect. Selectivity is the key. Better to find out earlier rather than later.  

Agree about retesting OSM-S-106. We could also make a couple of grams and make it freely available for anyone to test in their assay. Would be an interesting community effort to see how many assays it could be tested in? All data to be in the open.

**Comment by mbhebhe** [comment]
@cdsouthan @drc007 I have synthesized OSM-S-106 3 times so far and each batch has been retested at Dundee. 
@drc007  I will look into CA assays. I am open to doing an assay myself if there is an easy protocol that I can follow.

**Comment by cdsouthan** [comment]
Good, so a) keep at least some of all three batches back.  b) testing all 3 in /// at Dundee would be good c) also get assayed by at least one other center 

 (n.b. all these reproducibility results will strengthen the paper)

**Comment by mbhebhe** [comment]
@cdsouthan We have been assayed by Vicky Avery's lab a long time back, before I started on S3.

**Comment by drc007** [comment]
@mattodd Regarding OSM-S-106 selectivity, related but a bit off topic. Would there be merit in trying to establish a worldwide network of open-source assays that people could submit compounds to? Similar to CO-ADD but distributed over many labs with a central "results" page?

**Comment by cdsouthan** [comment]
@drc007 Certainly. This could define the variance(s) if centers were prepared to do enough true replicates (i.e. plot the Gaussian)  for the major stains and stages.  Ditto obviously for each target in vitro.  While we are about it we should somehow encourage Swiss-Prot to close the annotation of at least D7 so we had at least one well defined _Plasmodium_ proteome

**Comment by drc007** [comment]
@mattodd  @cdsouthan I was actually thinking more broadl, how about any assay for any target?

**Comment by cdsouthan** [comment]
Impeccable logic (as expected from @drc007)  which extrapolates to an open cross-screening consortium extending from organisms (as in the CO-ADD model) out to purified in vitro targets.  The (logical as ever) first step could  be to informally canvas groups who have published these, to see who's up for what.

**Comment by mattodd** [comment]
@drc007 indeed - a loose end captured in the [wiki](https://github.com/OpenSourceMalaria/Series3/wiki/Mechanism-of-Action), but where we still need experimental effort, right @mbhebhe ? (I was here just trying to summarise the present discussion in this issue, though). In fact I was just talking with someone here at Monash about this very issue, and will post about that elsewhere when I get a moment.

**Comment by rafacounago** [comment]
Hi - new to the discussion from SGC-UNICAMP,
We are running a program on infectious diseases down here and working on AaRSs. We would love to help as much as we can - our assay is up and running. We have access to the Pf protein but would need to obtain the compound and its negative control.
Cheers!

**Comment by mbhebhe** [comment]
@rafacounago welcome! I will email you and Caio to organise the shipping

**Comment by mattodd** [comment]
Closing. All this page captured in the wiki and new data will be explained in forthcoming paper.
